source/nodejs/adaptivecards-templating/src/template-engine.ts

```diff
@@ -47,6 +50,34 @@ class EvaluationContext {
     set $data(value: any) {
         this._$data = value;
     }
+
+    generateVersionJson() {
+        let pkg = require('./../package.json');
+        let version = pkg.version;
+        let versionSplit = version.split('.');
+
+        let build;
+        if (versionSplit[3]) {
+            build = versionSplit[3];
+        }
+
+        let patchSplit = versionSplit[2].split('-');
+
+        let suffix;
```

Comment:
nit: always assign default value if possible. For example `let suffix = undefined`